webapp/channels/src/packages/mattermost-redux/src/actions/users.ts

```diff
@@ -344,7 +344,6 @@ export function getProfilesWithoutTeam(page: number, perPage: number = General.P
 }
 
 export enum ProfilesInChannelSortBy {
-    None = '',
```

Comment:
Could you revert the changes to these enums? While they might not be used now, some of them seem like things we might use in the future or that they're default values that aren't explicitly referenced but are the default case in if-else or switch blocks